UM/Qt/Bindings/ProfilesModel.py

```diff
@@ -54,9 +61,22 @@ def setAddUseGlobal(self, add):
     def addUseGlobal(self):
         return self._add_use_global
 
+    addWorkingProfileChanged = pyqtSignal()
+
+    def setAddWorkingProfile(self, add):
+        if add != self._add_working_profile:
+            self._add_working_profile = add
+            self._onProfilesChanged()
+            self.addWorkingProfileChanged.emit()
```

Comment:
The ```WorkingProfile``` (working as in "Working Title", not as in "Working Man") is the profile each instance has attached to it, which the user changes by changing settings in the UI.

```addWorkingProfile``` is a flag in the profilesmodel whether or not to list this Working Profile to the listmodel, analogous from the ```addUseGlobal``` flag.

```addWorkingProfileChanged``` is the signal the fires when the ```addWorkingProfile``` flag is changed, analogous to the existing ```addUseGlobalChanged```.

Though I agree that the naming reads like a random concatenation of verbs (and a noun), it is in line with the existing ````AddUseGlobal```` naming.